Analysis/Tasks/Utils/include/AnalysisTasksUtils/UtilsDebugLcK0Sp.h

```diff
@@ -10,27 +10,54 @@
+  assert(listLabelsK0SPos.size() == listLabelsK0SPos.size());
```

Comment:
Assertions only have effect when compiled in debug mode, either have a normal `if` and throw an exception if it fails, or remove it.

Reply:
Done